doc/source/visualizing/plots.rst

```diff
@@ -343,46 +343,79 @@ Types of Projections
 """"""""""""""""""""
 
 There are several different methods of projections that can be made either
-when creating a projection with ds.proj() or when making a ProjectionPlot.
+when creating a projection with :meth:`~yt.static_output.Dataset.proj` or
+when making a :class:`~yt.visualization.plot_window.ProjectionPlot`.
 In either construction method, set the ``method`` keyword to be one of the
 following:
 
-``integrate`` (unweighted)
-    This is the default projection method. It simply integrates the
-    requested field  :math:`f(x)` along a line of sight  :math:`\hat{n}` ,
-    given by the axis parameter (e.g. :math:`\hat{i},\hat{j},` or
-    :math:`\hat{k}`).  The units of the projected field
-    :math:`g(X)` will be the units of the unprojected field  :math:`f(x)`
-    multiplied by the appropriate length unit, e.g., density in
-    :math:`\mathrm{g\ cm^{-3}}` will be projected to  :math:`\mathrm{g\ cm^{-2}}`.
+``integrate`` (unweighted):
```

Comment:
Not sure why but the `:math:` directive didn't render correctly in the build

![Screenshot 2022-09-05 at 09 20 27](https://user-images.githubusercontent.com/14075922/188391289-939f93e1-d35f-48d8-911a-680afabbfd80.png)

Reply:
Funny, this resolved OK for me on my laptop.

Reply:
probably nbd then, maybe the html artefacts are just broken